lockbox-ios/View/ItemListView.swift

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import WebKit
+
+class ItemListView : UITableViewController, ItemListViewProtocol {
+    var presenter:ItemListPresenter?
+    internal(set) var webView: WebView
+
+    private var items:[Item] = []
+
+    required init?(coder aDecoder: NSCoder) {
+        self.webView = WebView(frame: .zero, configuration: WKWebViewConfiguration())
+        super.init(coder: aDecoder)
+        self.presenter = ItemListPresenter(view: self)
+    }
+
+    func displayItems(_ items: [Item]) {
+        self.items = items
+        self.tableView.reloadData()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.addSubview(self.webView)
+        self.styleNavigationBar()
+
+        self.presenter?.onViewReady()
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return items.count
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "itemlistcell") as? ItemListCell else {
+            fatalError("couldn't find the right cell!")
+        }
+        let item = items[indexPath.row]
+
+        cell.titleLabel.text = item.title
+        cell.detailLabel.text = (item.entry.username == "" || item.entry.username == nil) ? "(no username)" : item.entry.username
+        cell.kebabButton.tintColor = UIColor.kebabBlue
+
+        return cell
+    }
+
+    private func styleNavigationBar() {
+        let prefButton = UIButton()
+        let prefImage = UIImage(named: "preferences")?.withRenderingMode(.alwaysTemplate)
+        prefButton.setImage(prefImage, for: .normal)
+        prefButton.tintColor = .white
+
+        self.navigationItem.rightBarButtonItem = UIBarButtonItem(customView: prefButton)
+        self.navigationItem.title = "Your Lockbox"
```

Comment:
what's the localization story for iOS?

Reply:
https://github.com/mozilla-lockbox/lockbox-ios/issues/89

I have been meaning to get this one uplifted from @sandysage to at least get the framework in place if not the translations themselves.